bin/tx-prover/src/proxy/mod.rs

```diff
@@ -254,12 +272,13 @@ static RATE_LIMITER: Lazy<Rate> = Lazy::new(|| Rate::new(Duration::from_secs(1))
 /// Request queue holds the list of requests that are waiting to be processed by the workers.
 /// It is used to keep track of the order of the requests to then assign them to the workers.
 pub struct RequestQueue {
```

Comment:
We should update the doc comments for this to mention the addition of the `Instant` to the request